File: mw_models/disk_test/make_init.py
import lilguys
import numpy as np

def main():
    p0 = 1
    v0 = 0.8
    r = np.array([8, 0, 0])

    v1 = np.array([0, v0, 0.05*v0])
    v2 = np.array([0, 1.2*v0, 0.05*v0])
    v3 = np.array([0, 0.8*v0, 0.05*v0])

    pos = np.array([r, -r, r])
    vel = np.array([v1, v2, v3])
    m = 0

    s = lilguys.Snapshot(pos, vel, m=m)
    s.save("disk_test.hdf5")


if __name__ == "__main__":
    main()

File: simulations/tests/hernquist/special_cases/make_init.jl
import LilGuys as lguys


function main()
    r0 = 2
    v0 = 0.5
    r = [r0, 0, 0]

    v1 = [0, v0, 0]
    v2 = 0.8*v1
    v3 = 1.2*v1

    pos = [r -r r]
    vel = hcat(v1, v2, v3)
    m = 0

    N = size(pos, 1)

    snap = lguys.Snapshot(pos, vel, zeros(N))
    lguys.save("initial.hdf5", snap)
end


if abspath(PROGRAM_FILE) == @__FILE__
    main()
end
